Core/scripts/dirac-install.py

```diff
@@ -2380,8 +2380,8 @@ def createBashrcForDiracOS():
       lines = ['# DIRAC bashrc file, used by service and agent run scripts to set environment',
                'export PYTHONUNBUFFERED=yes',
                'export PYTHONOPTIMIZE=x',
-               '[ -z "$DIRAC" ] && export DIRAC=%s' % proPath,
-               '[ -z "$DIRACOS" ] && export DIRACOS=$DIRAC/diracos',
+               'export DIRAC=%s' % proPath,
```

Comment:
That does not seem like a good idea to me. Why do you want to do that ?

Reply:
Why it's not a good idea?

My argument is that, in case there are 2 existing installations (e.g. the first in directory `/dir1` with version `X`, and a second in `dir2` with version `Y`) sourcing one bashrc after the other would end up in a possible disaster.

Reply:
yes but doing that is clearly looking for trouble. Even more: the two diracosrc would be sourced, where there is no such protection, so it would be unusable anyway. Plus managing one DIRAC installation is not easy, having several on the system is suicide :D 

My arguments for keeping it as it is: being able to test DIRAC with a different version of DIRACOS and vice versa 

What triggers this change anyway ?